### ISSUE TITLE
flutter_stripe_web fails to build (missing implementation handleNextAction)

### ISSUE THREAD

**Issue by mauriziopinotti** [flutter-stripe/flutter_stripe#755]
**Describe the bug**
As per title, flutter_stripe_web 1.0.0 fails to build with flutter_stripe 3.0.2.

**To Reproduce**
1. Use these dependencies in `pubspec.yaml`:

```yaml
  flutter_stripe: ^3.0.2
  flutter_stripe_web: ^1.0.0
  stripe_checkout: ^1.0.0
```

2. Build for Web
3. Observe a failure with exception `The non-abstract class 'WebStripe' is missing implementations for these members: StripePlatform.handleNextAction`

**Additional context**
Full log:

```
[+3591 ms] ../../../.pub-cache/hosted/pub.dartlang.org/flutter_stripe_web-1.0.0/lib/src/web_stripe.dart:15:7: Error: The non-abstract class 'WebStripe' is missing implementations for these members:
[   +1 ms]  - StripePlatform.handleNextAction
[        ] Try to either
[        ]  - provide an implementation,
[        ]  - inherit an implementation from a superclass or mixin,
[        ]  - mark the class as abstract, or
[        ]  - provide a 'noSuchMethod' implementation.
[        ] class WebStripe extends StripePlatform {
[   +4 ms]       ^^^^^^^^^
[        ] ../../../.pub-cache/hosted/pub.dartlang.org/stripe_platform_interface-3.0.1/lib/src/stripe_platform_interface.dart:42:25: Context: 'StripePlatform.handleNextAction' is defined here.
[        ]   Future<PaymentIntent> handleNextAction(String paymentIntentClientSecret);
[        ]                         ^^^^^^^^^^^^^^^^
```

**Comment by mauriziopinotti** [comment]
Please notice it works on latest version from `master` so I guess you just forgot to upload the new version on pub.dev.

As a temporary workaround:

```yaml
  flutter_stripe_web:
    git:
      url: https://github.com/flutter-stripe/flutter_stripe.git
      path: packages/stripe_web
```

**Comment by glennarixxi** [comment]
It does not work on latest version of `stable` - `flutter_stripe ^3.0.2` `flutter_stripe_web ^1.0.0` - Work around above does work

**Comment by vscapital** [comment]
@glennarixxi hi.

Try this:

```
dependency_overrides:
  flutter_stripe_web:
    git:
      url: https://github.com/flutter-stripe/flutter_stripe
      path: packages/stripe_web
```

**Comment by jonasbark** [comment]
We've just released 1.0.1 with the fix

**Comment by mauriziopinotti** [comment]
> We've just released 1.0.1 with the fix

Thanks, confirmed working.
